src/components/Input/Input.tsx

```diff
@@ -237,7 +237,7 @@ const AdornmentButtonStyled = styled.button(({ theme, disabled }) => ({
       fill: theme.custom.colors.silverGray,
     },
   },
-  ":hover": {
+  ":hover:not(:disabled)": {
```

Comment:
openedx's hover styles were overriding our `:hover` b/c theirs used `:hover:not(:disabled)` which is more specific. Changing ours to  `:hover:not(:disabled)` because then ours is more specific...but also because that's how ours should have been in the first place.